srcpkgs/ksh/patches/fix-cross.diff

```diff
@@ -0,0 +1,9 @@
+--- scripts/libast_prereq.sh.orig	2021-03-31 22:45:26.222481811 +0200
++++ scripts/libast_prereq.sh	2021-03-31 22:45:34.162481713 +0200
+@@ -25,5 +25,5 @@ cd "$MESON_BUILD_ROOT"
+ # Generate header files whose content depends on the current platform.
+ "$MESON_SOURCE_ROOT/scripts/siglist.sh" > features/siglist.h
+ # shellcheck disable=SC2086
+-$CC -D_BLD_DLL $INC_DIRS -std=gnu99 -o "sfinit" "$MESON_SOURCE_ROOT/etc/sfinit.c"
++cc -D_BLD_DLL $INC_DIRS -std=gnu99 -o "sfinit" "$MESON_SOURCE_ROOT/etc/sfinit.c"
```

Comment:
Is this safe?

>  Generate header files whose content depends on the current platform.

Worries me slightly :/

Reply:
You're right. I thought the build just relied on ast_config.h, but there are other compiler macros used.